cloud/google/machineactuator.go

```diff
@@ -719,6 +705,28 @@ func (gce *GCEClient) getImagePath(img string) (imagePath string) {
 	return defaultImg
 }
 
+func newDisks(config *gceconfigv1.GCEProviderConfig, zone string, imagePath string, minDiskSizeGb int64) []*compute.AttachedDisk {
+	var disks []*compute.AttachedDisk
+	for idx, disk := range config.Disks {
+		diskSizeGb := disk.InitializeParams.DiskSizeGb
+		if diskSizeGb < minDiskSizeGb {
+			glog.Info("increasing disk size to %$v gb, the supplied disk size of %v gb is below the minimum", minDiskSizeGb, diskSizeGb)
```

Comment:
Is the %$v intentional?

Reply:
(nit)

Reply:
Not a nit -- definitely a mistake. Will correct as well as rebase on top of the latest changes (they look significant).

Reply:
Ah. In that case, I will withdraw my approval so the PR dashboard will tell me to take a look after the rebase.